src/www/util.js

```diff
@@ -1,16 +1,25 @@
-window.Ionic = window.Ionic || {};
-window.Ionic.WebView = window.Ionic.WebView || {};
+var exec = require('cordova/exec');
 
-window.Ionic.WebView.convertFileSrc = window.Ionic.WebView.convertFileSrc || function(url) {
-  if (!url) {
-    return url;
+var WebView = {
+  convertFileSrc: function(url) {
+    if (!url) {
+      return url;
+    }
+    if (!url.startsWith('file://')) {
+      return url;
+    }
+    url = url.substr(7); // len("file://") == 7
+    if (url.length === 0 || url[0] !== '/') { // ensure the new URL starts with /
+      url = '/' + url;
+    }
+    return window.WEBVIEW_SERVER_URL + '/_file_' + url;
+  },
+  setServerBasePath: function(path) {
+    exec(null, null, 'IonicWebView', 'setServerBasePath', [path]);
+  },
+  getBasePath: function(callback) {
```

Comment:
Can we do either `setServerBasePath()` and `getServerBasePath()` OR `setBasePath()` and `getBasePath()` ? Just for consistency.

Reply:
Sure. @nphyatt any preference?